File: Assets/Scripts/misc/MainMenuEffects.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public class MainMenuEffects : MonoBehaviour
{
    public ParticleSystem customParticleSystem;
    public virtual void Start()
    {
        if (this.customParticleSystem != null)
        {
            this.customParticleSystem.Simulate(10);
        }
    }

}

File: Assets/Scripts/misc/MainMenuEffects.js
#pragma strict
#pragma implicit
#pragma downcast

class MainMenuEffects extends MonoBehaviour
{
    public var customParticleSystem : ParticleSystem;

    function Start()
    {
        if (customParticleSystem != null)
        {
            customParticleSystem.Simulate(10);
        }
    }
}
